templates/ci/ant.yml

```diff
@@ -0,0 +1,27 @@
+# Ant
+# Build your Java projects and run tests with Apache Ant.
+# https://aka.ms/yaml
+
+queue: 'Hosted VS2017'
+
+#variables:
+#  system.debug: 'true'
+
+steps:
+- task: Ant@1
+  displayName: 'Build with Ant'
+  inputs:
+    buildFile: 'build.xml'
+
+- task: CopyFiles@2
+  displayName: 'Copy artifacts'
+  inputs:
+    contents: '**/*.jar'
+    targetFolder: '$(build.artifactStagingDirectory)'
+
+- task: PublishBuildArtifacts@1
```

Comment:
@ericsciple : Chris mentioned there will be a shortcut for this. It will be good to change this when that is available.

Reply:
definitely, i'll keep this in mind. we're working with RM to define the syntax for modeling releases and artifacts. I think we'll use something like `download`, so maybe the reverse `upload` would make sense. or maybe `publish` would just be more intuitive...

Reply:
Just 2-cents: "Publish" aligns with the "Publish Artifacts" task name, but that name sounds like a package management action.  Maybe it makes sense to keep the name "publish" if the arguments/inputs are similar to the task, but otherwise "upload" sounds good and correlates with "download" like you say.